ios/history.md

```diff
@@ -1,5 +1,8 @@
 # Keyman for iPhone and iPad Version History
 
+## 2018-08-02 10.0.208 stable
+* Removed unnecessary UIFileSharingEnabled flag that caused Keyman to be rejected (#1087)
```

Comment:
This is okay but a little alarming to end users perhaps. I suggest stating just 'Removed unnecessary UIFileSharingEnabled flag' -- the details can be followed for the technically minded.